Resources/Private/Scss/extra.scss

```diff
@@ -0,0 +1,114 @@
+$grid-s-width: 312px;
+$grid-m-width: 768px;
+$grid-l-width: 960px;
+
+$media-s: "(max-width: 767px)";
+$media-m: "(min-width: 768px)";
+$media-l: "(min-width: 1024px)";
```

Comment:
There seems to be some inconsistency between media-l and grid-l-width. Why is one 1024 and the other 960?

Reply:
It's just the decision when a certain media query triggers a certain grid. could be the same, doesn't have to be. They are the same in your grid system?

Reply:
I use 768 and 960px, as in https://github.com/subugoe/aac-frontend-docu/blob/master/views_and_grids.md described. 
For the rest or the code, L starts with 961px, S and M are the same. 
We should try to use the same breakpoints.